README.md

```diff
@@ -171,6 +171,8 @@ EXAMPLE
   Login as HMD successfully!
 ```
 
+Alternatively, you may supply an email and password via the `CMD_EMAIL` and `CMD_PASSWORD` environment variables, respectively.
```

Comment:
There's a configuration table in the above section. Could you please also add these two keys to it?